src/Data/Codec/JSON/Strict.purs

```
@@ -0,0 +1,172 @@
+module Data.Codec.JSON.Strict
+  ( PropCodec
+  , ClaimedProps
+  , objectStrict
```

Could just call this `object` now I suppose?

---

Then there is a high chance of mixing up the two in consumer code, and there won't be any warning, the code will just silently accept unknown fields.